tests/Engagement_unit_test.py

```diff
@@ -66,22 +69,6 @@ def test_edit_created_new_engagement(self):
         EngagementTXT = driver.find_element_by_tag_name("BODY").text
         self.assertTrue(re.search(r'Engagement updated successfully.', EngagementTXT))
 
-    def test_engagement_import_scan_result(self):
```

Comment:
what's wrong with that section? not working or not interesting? if just not working, could you leave it only commented out so that someone can work on it later it they find it's actually an interesting test? (or rather: leave those lines and comment out the call `suite.addTest(EngagementTest('test_engagement_import_scan_result'))`

Reply:
exactly the same test exist in finding_unit_test.py file ... **test_import_scan_result**